src/services/game.ts

```diff
@@ -106,6 +116,9 @@ export class GameService {
       });
 
       board.elapsedTicks++;
+      game.gameStatus.elapsedTicks = board.elapsedTicks;
+      game.gameStatus.foodLeft = board.food.length;
+      game.gameStatus.gameLength = MAX_TICKS;
```

Comment:
Does this need to be set again?

Reply:
this is going to be refactored a bit coming up in the next batch of PR's, the updateboard logic doesn't have access to the whole game object, so i just duplicated this elapsedTicks property for the time being.